requirements.txt

```
@@ -2,4 +2,4 @@ requests>=2.23.0
 six>=1.15.0
 pytest
 delayed-assert
-enum34
+enum34; python_version <= '3.3'
```

aenum is compatible with both,
```suggestion
aenum
```

---

Very true - figured you would be more inclined to keep the dependency the same, will update my commit now. Thanks @iivanou